src/styles/index.scss

```diff
@@ -1,3 +1,84 @@
-body {
-  margin: 0;
+.body {
+  border: 1px solid red;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+
+  /* box-sizing: border-box; */
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  background-color: #fff;
+  position: relative;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+}
+
+.innerD {
+  width: 5vmin;
+  height: 5vmin;
+  background-color: #f6a603;
+  border-radius: 50%;
+  position: relative;
+  z-index: 2;
+}
+
+.stopwatch__minutes {
+  width: 3vmin;
+  height: 40vmin;
+  background-image: linear-gradient(180deg, #0700ff, transparent, transparent);
+  position: absolute;
+  animation-name: move;
+  animation-duration: 3600s;
+  animation-timing-function: steps(60, end);
+  animation-delay: 0s;
+  animation-iteration-count: 30;
+  animation-direction: normal;
+
+  /* transform: translateY(-50%); */
+
+  /* transform-origin: bottom center; */
+}
+
+@keyframes move {
+  0% {
+    transform: /* translateY(-50%) */ rotate(0);
+  }
+  100% {
+    transform: /* translateY(-50%) */ rotate(360deg);
+  }
+}
+
+.stopwatch__seconds {
+  width: 1.5vmin;
+  height: 76vmin;
+  background-color: transparent;
+  background-image: linear-gradient(180deg, #2c8000, transparent, transparent);
+  position: absolute;
+
+  /* transform: translateY(-50%); */
+
+  /* transform-origin: bottom center; */
+  animation-name: move;
+  animation-duration: 60s;
+  animation-timing-function: linear;
+  animation-delay: 0s;
+  animation-iteration-count: 30;
+  animation-direction: normal;
+}
+
+.stopwatch--speed-up {
+  & .watch__minutes {
+    animation-duration: 600s;
+  }
+
+  & .stopwatch__seconds {
+    animation-duration: 10s;
+  }
```

Comment:
The BEM modifier should be named with a double dash and should follow the element it modifies. For example, '.stopwatch__minutes--speed-up' instead of '.stopwatch--speed-up .watch__minutes'.